README.md

```diff
@@ -21,11 +21,47 @@ Differences are reported as one or more change records. Change records have the
     * `N` - indicates a newly added property/element
     * `D` - indicates a property/element was deleted
     * `E` - indicates a property/element was edited
-    * `A` - indicates a change occurred within an array
     * `C` - indicates a conflicting change was made to both mine and theirs
 * `path` - the property path (from the parent)
-* `parent` - the value on parent (undefined if kind === 'A')
+* `parent` - the value on parent
 * `theirs` - the value on theirs (undefined if kind === 'N')
 * `mine` - the value on mine (undefined if kind === 'D')
-* `index` - when kind === 'A', indicates the array index where the change occurred
-* `item` - when kind === 'A', contains a nested change record indicating the change that occurred at the array index
\ No newline at end of file
+
+### Options
+Options object can be used to denote certain flags.  Object should be replica of `mine` with flags set on specific keys
+#### Example
+
+Mine:
+````
+{ 
+  key1: { 
+    childKey1: { 
+      key: value
+    },
+    childKey2: value
+  },
+  key2: {
+    childKey1: value
+  }
+}
+````
+
+Options:
+````
+{ 
+  key1: { 
+    childKey1: { 
+      ignoreKey: true 
+    },
+    ignoreOrder: true, 
+  },
+  key2: {
+    ignoreKey: true
+  }
+}
+````
+
+Flags
+* `ignoreKey` - Do not diff this key
```

Comment:
It won't let me leave a comment on that line, but Line 12 TODO should be removed now that this is done